src/openrct2/Limits.h

```diff
@@ -8,16 +8,19 @@
  *****************************************************************************/
 
 #pragma once
+
 #include "rct2/Limits.h"
 
+#include <limits>
+
 namespace OpenRCT2::Limits
 {
     constexpr uint16_t kMaxRidesInPark = 1000;
     constexpr uint16_t kMaxStationsPerRide = 255;
     constexpr uint8_t kCustomerHistorySize = RCT12::Limits::kCustomerHistorySize;
     constexpr uint16_t kMaxGolfHoles = RCT12::Limits::kMaxGolfHoles;
     constexpr uint16_t kMaxHelices = RCT12::Limits::kMaxHelices;
```

Comment:
These are also `31`, should we take the chance to update them as well?

Reply:
I found out that helices are stored on the last 5 bits of `special_track_elements`, so they shouldn't be changed arbitrarily because everything there might break, therefore I added 9a455774dd4e7f7970dcff6cd8b18f73170be07d to warn if anyone wants to change it.

In any case, it seems that for ride ratings they are never shown explicitly, nor accounted for when over 15
https://github.com/OpenRCT2/OpenRCT2/blob/930e215103752fec1f6f02c1ce25a6805e9c4efb/src/openrct2/ride/RideRatings.cpp#L1588-L1595

Reply:
As for golf holes, I removed the cap since they are stored separately. That does make it possible to theoretically infinitely boost the excitement of a mini golf:
https://github.com/OpenRCT2/OpenRCT2/blob/930e215103752fec1f6f02c1ce25a6805e9c4efb/src/openrct2/ride/RideRatings.cpp#L2031-L2035
But it is such a tiny bump compared to the other bonuses, that I think we're fine.
https://github.com/OpenRCT2/OpenRCT2/blob/930e215103752fec1f6f02c1ce25a6805e9c4efb/src/openrct2/ride/rtd/gentle/MiniGolf.h#L63-L69